src/lighteval/metrics/metrics.py

```diff
@@ -522,13 +522,20 @@ def higher_is_better():
         return res
 
     @staticmethod
-    def corpus_level_fns() -> dict[str, callable]:
+    def corpus_level_fns(metrics: list[str]) -> dict[str, callable]:
```

Comment:
Before this fix, we were overwritting the aggregations! (which was never a problem since we tend to use the same aggregation for similar evals, like np.mean for acc)